README.md

```diff
@@ -65,6 +141,7 @@ To run local tests, use:
 ```
 make test-sandbox
 ```
+If a lot of the tests fail and the sandbox crashes with Cannot read properties of null (reading 'statusCode')  you may not be using the correct version of node. Check the version being used by that instance of the terminal and run re-run nvm use lts/fermium if required
```

Comment:
```suggestion
If a lot of the tests fail and the sandbox crashes with
``` Cannot read properties of null (reading 'statusCode') ```
you may not be using the correct version of node. Check the version being used by that instance of the terminal and run re-run nvm use lts/fermium if required
```
Also, just double checking on `fermium`/`gallium`

Reply:
see previous comment on this, for time being, lts/fermium is the one to go for